katello/backup.rb

```diff
@@ -28,10 +28,20 @@ def initialize(foreman_proxy_content, foreman_proxy, program, accepted_scenarios
       setup_opt_parser
     end
 
+    # do not use run_cmd() inside this method, to avoid a recursive loop
```

Comment:
Do you mind being a bit more verbose and say you aren't using run_cmd inside cleanup method because run_cmd calls cleanup and will cause an infinite loop? (I only ask because I was confused to where the loop would come from at first)

Reply:
done!